src/bindings/python/src/pyopenvino/pyopenvino.cpp

```diff
@@ -69,6 +69,24 @@ std::string get_version() {
 
 PYBIND11_MODULE(pyopenvino, m) {
     m.doc() = "Package openvino.pyopenvino which wraps openvino C++ APIs";
+#ifdef CI_BUILD_NUMBER
+    std::string pyopenvino_version = CI_BUILD_NUMBER;
+    std::string runtime_version = get_version();
+    bool is_custom_pyopenvino_version = pyopenvino_version.find("custom_") == 0;
+    bool is_custom_runtime_version = runtime_version.find("custom_") == 0;
+    if (!is_custom_pyopenvino_version && !is_custom_runtime_version) {
+        auto versions_compatible =
```

Comment:
Looks like we don't need this local variable :)
Moreover we can move condition from 77 line to assert.

Reply:
I just wanted to make assert message more user friendly. With current approach message header will be `Check 'version_compatible' failed at pyopenvino.cpp:87`. With huge condition inside assert , message will be `Check 'is_custom_pyopenvino_version || !is_custom_runtime_version || pyopenvino_version.empty() || runtime_version.empty() || pyopenvino_version == runtime_version' failed at pyopenvino.cpp:87` which is much less readable (even though there is a detailed explanation after this ugly header)